README.md

```diff
@@ -200,17 +200,40 @@ Properties passed to the `w` function represent the public API for a widget.
 
 The properties lifecycle starts when properties are passed to the widget.
 The properties lifecycle is performed in the widgets `setProperties` function.
-This function uses the widget instance's `diffProperties` function to determine whether any of the properties have changed since the last render cycle.
-By default `diffProperties` provides a shallow comparison of the previous properties and new properties.
 
-The `diffProperties` function is also responsible for creating a copy (the default implementation uses`Object.assign({}, newProperties)` of all changed properties.
-The depth of the returned diff is equal to the depth used during the equality comparison.
+This function uses the widget's `diffProperty` and `diffProperties` decorators to determine whether any of the properites have changed since the last render cycle.
```

Comment:
first off, thanks for being awesome and updating the readme 😄. do you think somewhere we should state in bold lettering that by default - changing a property will not cause a change/re-render, without adding either of those decorators?